src/cli/dev.test.ts

```diff
@@ -102,7 +102,7 @@ describe("dev command", () => {
 
   it("should run dev command successfully", async () => {
     const writeFile = fs.writeFile as Mock;
-    await program.parseAsync(["node", "test", "dev", "--yes"]);
+    await program.parseAsync(["npx", "pepr", "dev", "--yes"]);
```

Comment:
```suggestion
    await program.parseAsync(["dev", "--yes"], { from: "user" });
```
Consistency with other tests we've written, though it's functionally equivalent

Reply:
This breaks the test

Reply:
well i think let me check, I tried to make it like that

Reply:
oh i see, i was missing `{from: "user"}`